akka-cluster-sharding/src/test/scala/akka/cluster/sharding/LeastShardAllocationStrategySpec.scala

```diff
@@ -23,61 +42,76 @@ class LeastShardAllocationStrategySpec extends AkkaSpec {
       regionC -> shards.takeRight(cCount).toVector)
   }
 
+  private def startedAllocationStrategy(rebalanceThreshold: Int, maxSimultaneousRebalance: Int) = {
+    val allocationStrategy = new LeastShardAllocationStrategy(rebalanceThreshold, maxSimultaneousRebalance)
+    allocationStrategy.start(system)
+    // FIXME why do I need this here or else some test fails not seeing any members?
+    awaitAssert(allocationStrategy.members.size should ===(1))
+    allocationStrategy
+  }
+
   "LeastShardAllocationStrategy" must {
+    "create a one node cluster" in {
+      // needed for the rest of the tests (looks at cluster state)
+      val cluster = Cluster(system)
```

Comment:
Would be nice if we could have these unit tests without `Cluster` extension. That the strategy has a `def currentClusterState` that can be overridden in test or some injection of accessor function to the CurrentClusterState.

Reply:
I'll look into doing that.

Reply:
Fixed